libasn1compiler/asn1c_misc.c

```diff
@@ -61,8 +61,13 @@ asn1c_make_identifier(enum ami_flags_e flags, asn1p_expr_t *expr, ...) {
 		/*
 		 * Estimate the necessary storage size
 		 */
-		if(expr->Identifier == NULL)
-			return "Member";
+		if(expr->Identifier == NULL) {
+			if(storage_size < 10) {
+				storage = calloc(1, 20);  /* reserve space for 'Memberxxx' */
+			}
+			sprintf(storage, "Member%d", expr->_type_unique_index);
```

Comment:
Only do this if `storage != NULL`. Also I would suggest `snprintf(storage, storage_size, ...` for safety.

Reply:
I meant to use realloc, but .......

Thanks for the advice to replace it with snprintf()